src/config.txt

```diff
@@ -0,0 +1,5 @@
+[credentials]
+CONSUMER_KEY = <CONSUMER_KEY>
+CONSUMER_SECRET = <CONSUMER_SECRET>
+ACCESS_KEY = <ACCESS_KEY>
+ACCESS_SECRET = <ACCESS_SECRET>
```

Comment:
I get the point of using a placeholder in this case, but I don't think that it's not a good idea to assign values that are incorrect by default.

Reply:
I understand your point, should I assign a NULL value to the variables initially?

Reply:
Yeah, something like `CONSUMER_KEY =`! :D